shared/src/utils/random.ts

```diff
@@ -57,6 +66,35 @@ export default class Random {
     return Math.floor(this.float(min, max + 1))
   }
 
+  /**
+   * Returns a random integer (normal distribution)
+   *
+   * The function tries to generate a number within the range of min and max.
+   * If it fails to do so after 10 tries, it returns the mean value.
```

Comment:
Does it make sense to check if the min/max interval is within the 2 oder 3 standard deviation interval?

Reply:
I agree that it should only be used when the min/max falls within 2 to 3 standard deviations, but this check should be performed at the point where the function is called. Otherwise, the function’s output may not be as reliable.
So I would not add a `if => throw Error`, if that's okay?

Reply:
At least during development there should be an error if the chosen parameter do not produce the right distribution. Another way would be to remove min and max parameters and leave the handling of this case to the user of the function.

Reply:
`if (mean - 3 * stdev < min || mean + 3 * stdev > max) { throw ... }` I added this check and a note in the docstring.

Reply:
Looks good.